### PR TITLE
[arci-speak-audio][openrr-apps] Add audio speaker to openrr apps

### PR DESCRIPTION
**Pull request by sktometometo**
- [x] ~~change the argument of AudioSpeaker::new() to a reference~~
- [x] change hashmap type of AudioSpeaker from HashMap<String,String> to HashMap<String,PathBuf>
- [x] rename create_speaker_* methods with explicit Speaker type names.
- [x] add AudioSpeaker to openrr-apps
- [x] add examples of config files with AudioSpeaker

### CODE REVIEW

File: arci-speak-audio/src/lib.rs [hunk 1]
```diff
@@ -25,9 +25,9 @@ pub struct AudioSpeaker {
 
 impl AudioSpeaker {
     /// Creates a new `AudioSpeaker`.
-    pub fn new(hashmap: HashMap<String, String>) -> Self {
+    pub fn new(hashmap: &HashMap<String, String>) -> Self {
         Self {
-            message_to_file_path: hashmap,
+            message_to_file_path: hashmap.clone(),
         }
```
Comment (OTL) [inline arci-speak-audio/src/lib.rs:31]: I think `before` version is better.

Comment (sktometometo) [reply arci-speak-audio/src/lib.rs:31]: reverted to the `before` version

**Review by OTL — APPROVED** [review]

**Comment by OTL** [comment]
I think arci-* crates can not depend on openrr-client (as a design).
We have to move the resolve function out of arci-speak-audio.

**Comment by OTL** [comment]
@sktometometo ちょっとややこい感じになってたんでスカッシュしちゃってちょっと直しました。

**Comment by sktometometo** [comment]
了解です、ありがとうございます

**Pull request merged**